concent_api/core/tests/test_core_views.py

```diff
@@ -370,15 +370,16 @@ def test_send_should_reject_message_when_timestamp_too_far_in_future(self):
 
         with freeze_time("2017-11-17 10:00:00"):
             timestamp = self._create_datetime_from_string("2017-11-17 10:10:00")
-            assert timestamp - parse_timestamp_to_utc_datetime(get_current_utc_timestamp()) > golem_settings.FUTURE_TIME_TOLERANCE
+            assert timestamp - parse_timestamp_to_utc_datetime(
```

Comment:
We don't use this style of line breaking.

Reply:
Done